packages/kit/src/vite/index.js

```diff
@@ -286,6 +286,9 @@ function kit() {
 			log.info(`Client build completed. Wrote ${chunks.length} chunks and ${assets.length} assets`);
 
 			log.info('Building server');
+			const assets_prefix = `${svelte_config.kit.paths.assets || svelte_config.kit.paths.base}/${
```

Comment:
`paths.assets` is a URL: https://kit.svelte.dev/docs/configuration#paths

some places you're using the prefix as a directory and some places you're using it as a URL. it's not going to work as currently written. e.g. you're copying files to the prefix, which doesn't work when `assets` is provided

Reply:
Hm, not quite sure why I had that in there to begin with. Got rid of it.